zh/restore-from-pv-using-br.md

```diff
@@ -0,0 +1,145 @@
+---
+title: 使用 BR 工具恢复持久卷上的备份数据
+summary: 介绍如何使用 BR 工具将存储在持久卷上的备份数据恢复到 TiDB 集群
+---
+
+# 使用 BR 工具恢复持久卷上的备份数据
+
+本文描述了如何将存储在[持久卷](https://kubernetes.io/zh/docs/concepts/storage/persistent-volumes/)上的备份数据恢复到 Kubernetes 环境中的 TiDB 集群。底层通过使用 [`BR`](https://docs.pingcap.com/zh/tidb/dev/backup-and-restore-tool) 来进行集群恢复。
+
+本文描述的持久卷指任何 [Kubernetes 支持的持久卷类型](https://kubernetes.io/zh/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes)。以下示例以 NFS 存储卷为例，介绍如何将存储在持久卷上指定路径的集群备份数据恢复到 TiDB 集群。
+
+本文使用的恢复方式基于 TiDB Operator 新版（v1.1 及以上）的 CustomResourceDefinition (CRD) 实现。
+
+## 数据库账户权限
+
+- `mysql.tidb` 表的 `SELECT` 和 `UPDATE` 权限：恢复前后，restore CR 需要一个拥有该权限的数据库账户，用于调整 GC 时间
```

Comment:
Note for versions that do not need to GC.

Reply:
fixed